sdk/tests/conformance/textures/misc/tex-video-using-tex-unit-non-zero.html

```diff
@@ -0,0 +1,218 @@
+<script id="shader-fs" type="x-shader/x-fragment">
+    precision mediump float;
+
+    uniform sampler2D uTexture0;
+    uniform sampler2D uTexture1;
+
+    varying vec2 vUV;
+
+    void main(void) {
+        vec4 texture0Color = texture2D(uTexture0, vUV);
+        vec4 texture1Color = texture2D(uTexture1, vUV);
+
+        gl_FragColor = mix(texture0Color, texture1Color, 0.5);
+    }
+</script>
+<script id="shader-vs" type="x-shader/x-vertex">
+    attribute vec3 aVertexPosition;
+    attribute vec2 aTextureCoord;
+
+    varying vec2 vUV;
+
+    void main(void) {
+        vUV = aTextureCoord;
+        gl_Position = vec4(aVertexPosition, 1.0);
+    }
+</script>
+
+<script>
+"use strict";
+
+var wtu = WebGLTestUtils;
+description("Regression test for crbug.com/676719");
+
+var gl = wtu.create3DContext("example", {preserveDrawingBuffer: true});
+
+var program = wtu.setupProgram(gl, ["shader-vs", "shader-fs"],
+                               ["aVertexPosition", "aTextureCoord"], [0, 1]);
+
+if (!gl || !program) {
+  testFailed("Test setup failed");
+}
+
+var loc0 = gl.getUniformLocation(program, "uTexture0");
+var loc1 = gl.getUniformLocation(program, "uTexture1");
+if (!loc0 || !loc1) {
+  testFailed("Failed to query uniform locations");
+}
+gl.uniform1i(loc0, 0);
+gl.uniform1i(loc1, 1);
+
+wtu.setupUnitQuad(gl, 0, 1);
+
+// Load textures
+var imageElement = document.getElementById("src-image");
+var resourcePath = "../../../resources/";
+
+var videos = [
+  { src: resourcePath + "red-green.mp4",
+    type: 'video/mp4; codecs="avc1.42E01E, mp4a.40.2"' },
+  { src: resourcePath + "red-green.webmvp8.webm",
+    type: 'video/webm; codecs="vp8, vorbis"' },
+  { src: resourcePath + "red-green.bt601.vp9.webm",
+    type: 'video/webm; codecs="vp9"' },
+  { src: resourcePath + "red-green.theora.ogv",
+    type: 'video/ogg; codecs="theora, vorbis"' },
+];
+
+function runAllTests() {
+  return new Promise(function(resolve, reject) {
+    var videoNdx = 0;
+    var video;
+    function runNextVideo() {
+      if (video) {
+        video.pause();
+      }
+      if (videoNdx == videos.length) {
+        resolve("SUCCESS");
+        return;
+      }
+      var info = videos[videoNdx++];
+      debug("");
+      debug("testing: " + info.type);
+      video = document.createElement("video");
+      var canPlay = true;
+      if (!video.canPlayType) {
+        testFailed("video.canPlayType required method missing");
+        runNextVideo();
+        return;
+      }
+      if(!video.canPlayType(info.type).replace(/no/, '')) {
+        debug(info.type + " unsupported");
+        runNextVideo();
+        return;
+      };
+      document.body.appendChild(video);
+      video.type = info.type;
+      video.src = info.src;
+      wtu.startPlayingAndWaitForVideo(video, runTest);
+    }
+    function runTest() {
+      var texture0 = gl.createTexture();
```

Comment:
Should we pause the current video here, so that we're guaranteed that the two texImage2D calls taking the video element receive the same data?